src/components/CardNote.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { styled } from "@mui/system";
+import PropTypes from 'prop-types';
+
+import DATA_NOTES_LIST from "../notesData";
+
+const CardWrapper = styled("div")({
+  marginTop: 20,
+  marginBottom: 20,
+});
+
+const CardActiveColor = styled("div")(({ activeColor }) => ({
+  color: activeColor,
+}));
+
+export default function CardNote ({
+  note: { id, title, description, date },
+  activeNote,
+  setActiveNote,
+}) {
+  const [color, setColor] = useState(null);
+  const [display, setDisplay] = useState([]);
+
+  useEffect(() => {
+    if (activeNote !== id) {
+      setColor(null);
+      setDisplay(DATA_NOTES_LIST);
+    }
+  }, [activeNote, id]);
+
+  const handleColor = () => {
+    const newColor = color === "#00BFFF" ? null : "#00BFFF";
+    const newId = activeNote === id ? null : id;
```

Comment:
Styled components allow to send props inside. Use this opportunity please, instead this function and strings